internal/testrunner/runners/system/tester.go

```diff
@@ -428,6 +428,11 @@ func (r *tester) createAgentInfo(policy *kibana.Policy, config *testConfig, runI
 		info.Agent.User = "root"
 	}
 
+	// This could be removed once package-spec adds this new field
+	if !slices.Contains([]string{"", "default", "complete"}, info.Agent.BaseImage) {
+		return agentdeployer.AgentInfo{}, fmt.Errorf("invalid value for agent.base_image")
```

Comment:
Nit. Include the value?
```suggestion
		return agentdeployer.AgentInfo{}, fmt.Errorf("invalid value for agent.base_image: %q", info.Agent.BaseImage)
```